app/mailers/kudo_mailer.rb

```diff
@@ -41,7 +41,7 @@ def batch_kudo_notification(user_id, user_kudos)
       end
       mail(
         to: user.email,
-        subject: t('mailer.kudos.you_have', app_name: ArchiveConfig.APP_SHORT_NAME)
+        subject: default_i18n_subject(app_name: ArchiveConfig.APP_SHORT_NAME)
```

Comment:
Does this work okay? I don't think we have a test for it, and the kudo mailer keys are not named with the [expected `[mailer_scope, action_name]` pattern](https://apidock.com/rails/ActionMailer/Base/default_i18n_subject). #4100 adds a test for the subject line and renames the keys, so it might be easier to leave this out for now unless we're sure it works.

Reply:
Noting for self, changes in kudo mailer are removed in my latest draft.